content/pt-br/docs/reference/using-api/client-libraries.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Bibliotecas de clientes
+content_type: conceito
+weight: 30
+---
+
+<!-- overview -->
+Esta página contém uma visão geral das bibliotecas clientes para usar a API do Kubernetes por meio de várias linguagens de programação.
+
+
+<!-- body -->
+Para desenvolver aplicações usando a [API REST do Kubernetes](/docs/reference/using-api/),
+você não precisa implementar as chamadas da API e os tipos de solicitação/resposta. 
```

Comment:
Suggestion: in Portuguese, we generally use the word "requisição" instead of "solicitação" for HTTP request. Consider using "requisição" in this context (and others where it refers to an HTTP request).